spec/smoke/healthcheck_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "spec_helper_smoke"
+
+describe "Publish Teacher Training Smoke Tests", :aggregate_failures, smoke: true do
+  let(:base_url) { Settings.publish_url }
+
+  subject(:response) { HTTParty.get(url) }
+
+  describe "GET #{Settings.publish_url}/healthcheck" do
```

Comment:
Can we add a test for `/ping` as well?
- `/healthcheck` tells us something is wrong with the connection from publish to the api
- `/ping` tells us something is wrong with publish itself

Reply:
Do this `/ping` test needs to be back ported across?

Reply:
done, added ping/pong test